plugins/modules/virt_volume.py

```diff
@@ -0,0 +1,711 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, Maciej Delmanowski <drybjed@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: virt_volume
+author:
+    - Leonardo Galli (@galli-leo)
+short_description: Manage libvirt volumes inside a storage pool
+description:
+    - Manage I(libvirt) volumes inside a storage pool.
+options:
+    name:
+        aliases: [ "volume" ]
+        description:
+            - Name of the volume being managed. Note that the volume must be previously
+              defined with xml.
+        type: str
+    pool:
+        required: true
+        description:
+            - Name of the storage pool, where the volume is located.
+        type: str
+    state:
+        choices: [ "present", "absent", "undefined", "deleted" ]
+        description:
+            - Specify which state you want a volume to be in.
+              If 'present', ensure that the volume is present but do not change its
+              state; if it's missing, you need to specify xml argument.
+              If 'undefined' or 'absent', volume will be removed from I(libvirt) configuration (logically only!).
+              If 'deleted', volume will be wiped clean and then removed.
```

Comment:
```suggestion
            - If C(present), ensure that the volume is present but do not change its
              state; if it's missing, you need to specify xml argument.
            - If C(undefined) or C(absent), volume will be removed from I(libvirt) configuration (logically only!).
            - If C(deleted), volume will be wiped clean and then removed.
```
should we also explain a difference between undefined and absent?
If they do the same, should we leave only one of them?

Reply:
Yes, it seems to be totally the same. I'm gonna remove `undefined` since the other one sounds more fitting for the vocabulary of ansible.